django/test/runner.py

```diff
@@ -33,6 +33,12 @@
     tblib = None
 
 
+# Levels
+NOTSET = 0
+DEBUG = 10
+INFO = 20
```

Comment:
There's no need to define your own constants. You can just use `logging.INFO`, etc.

Reply:
Oh sure. I mentioned that there are few other places where self-defined constants were used, so I followed that. Removing it!